commit 95f4651a793030ec7ca60c636f4085f7d4dbffff
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-03-08T15:44:11+00:00 (7 months ago)
    
    MANTA-4161 more manta repositories should not use git+ssh:// URIs in package.json files
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/package.json b/package.json
index 52b98ff..d450502 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "bunyan": "0.21.4",
         "cmdutil": "1.1.0",
         "dashdash": "1.3.2",
-        "libmanta": "git+ssh://git@github.com:joyent/node-libmanta.git#master",
+        "libmanta": "https://github.com/joyent/node-libmanta.git#master",
         "moray": "3.3.0",
         "node-uuid": "1.4.0",
         "once": "1.2.0",
